sw/host/opentitanlib/src/transport/ti50/mod.rs

```diff
@@ -0,0 +1,55 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+use anyhow::Result;
+
+use std::cell::RefCell;
+use std::rc::Rc;
+
+use crate::io::gpio::GpioPin;
+use crate::io::spi::Target;
+use crate::io::uart::Uart;
+use crate::transport::{Capabilities, Capability, Transport};
+
+#[derive(Default)]
+pub struct HostEmulation {
+    pub port: u16,
+    _inner: RefCell<Inner>,
```

Comment:
I assume the underscore here is to stop the compiler from complaining that this member is never used and that you'll rename it as `inner` when the implementation is written.

Reply:
Right